pocs/camera/pyro.py

```diff
@@ -0,0 +1,475 @@
+import sys
+import os
+import glob
+import time
+from warnings import warn
+from threading import Event
+from threading import Timer
+from threading import Thread
+from threading import Lock
+import subprocess
+
+from astropy import units as u
+import Pyro4
+
+from pocs.base import PanBase
+from pocs.utils import current_time
+from pocs.utils import config
+from pocs.utils.logger import get_root_logger
+from pocs.utils import load_module
+from pocs.camera import AbstractCamera
+
+
+# Enable local display of remote tracebacks
+sys.excepthook = Pyro4.util.excepthook
+
+
+class Camera(AbstractCamera):
+    """
+    Class representing the client side interface to a distributed camera
+    """
+    def __init__(self,
+                 uri,
+                 name='Pyro Camera',
+                 model='pyro',
```

Comment:
Seems like we will also need to be able to specify the actual model behind the pyro proxy.

Reply:
The actual model behind the Pyro proxy is obtained from the proxy in `connect()`, and used to update the `model` property then. Consequently the value passed to `__init__()` is almost immediately discarded unless the attempt to connect to the remote camera fails.